main.js

```diff
@@ -0,0 +1,130 @@
+const myLibrary = [];
+
+const book = (author, title, pages, read) => {
+  const getBooks = () => myLibrary;
+
+  const addBookToLibrary = () => {
+    myLibrary.push({
+      author, title, pages, read,
+    });
+  };
+
+  return {
+    addBookToLibrary, author, title, pages, read, getBooks,
+  };
+};
+
+const ui = () => {
+  const addBookToUi = (book) => {
+    const list = document.getElementById('book-list');
+    // Create tr element
+    const row = document.createElement('tr');
+    // Insert cols
+
+    row.innerHTML = `
+        <td>${book.title}</td>
+        <td>${book.author}</td>
+        <td>${book.pages}</td>
+        <td><a href="#" class="change">${book.read}<a></td>
+        <td><a href="#" class="delete">X<a></td>`;
+
+    list.appendChild(row);
+  };
```

Comment:
It has a function that `loops through myLibrary`

Reply:
Kindly address this comment